chemreg/openapi/schemas/substance/synonym.libsonnet

```
@@ -0,0 +1,45 @@
+local source = import 'source.libsonnet';
+local substance = import 'substance.libsonnet';
+local synonym_quality = import 'synonymQuality.libsonnet';
+local synonym_type = import 'synonymType.libsonnet';
+
+ {
+  app: 'Substance',
+  type: 'synonym',
+  description: 'Everything about Synonyms.',
+  attributes: {
+    identifier: {
+      type: 'string',
+      maxLength: 1024,
+    },
+    qc_notes: {
+      type: 'string',
+      maxLength: 1024,
+      required: false,
+    },
+  },
+  relationships: [
+    {
+      object: source,
+      many: false,
+      default: 1,
+    },
+    {
+      object: substance + 
+        { 
+          relationships: [] 
+        },
```

```suggestion
      object: substance + 
        { 
          relationships: [] 
        },
      many: false,
      default: 1,
```
This is messing my import up for postman api.  It's probably fine to leave it the way it is if there is some reason not to change it.